components/shuttleInformation/shuttleScheduleService.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Should perhaps leave a comment that this is being disabled temporarily until the module that uses this variable will be created later. Should be deleted later